packages/mesos/buildinfo.json

```diff
@@ -1,24 +1,29 @@
 {
-  "requires": ["openssl", "libevent", "curl", "boost-libs"],
-  "single_source" : {
+  "requires": [
```

Comment:
Just white space changes here?

Reply:
The whitespace changes, here and below, are fixing the previous ad-hoc formatting.